### PR TITLE
Remove Result in Service

### PR DESCRIPTION
**Pull request by ruanwenjun**
## Purpose of the pull request

Use Result<Object>/Map<String, Object> as method result is not a good practice, this PR will refactor some service method.

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/AccessTokenController.java [hunk 1]
```diff
@@ -154,10 +155,10 @@ public Result queryAccessTokenList(@Parameter(hidden = true) @RequestAttribute(v
...
     @ResponseStatus(HttpStatus.OK)
     @ApiException(QUERY_ACCESSTOKEN_BY_USER_ERROR)
-    public Result queryAccessTokenByUser(@Parameter(hidden = true) @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
-                                         @PathVariable("userId") Integer userId) {
-        Map<String, Object> result = this.accessTokenService.queryAccessTokenByUser(loginUser, userId);
-        return this.returnDataList(result);
+    public Result<List<AccessToken>> queryAccessTokenByUser(@Parameter(hidden = true) @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
+                                                            @PathVariable("userId") Integer userId) {
+        List<AccessToken> accessTokens = accessTokenService.queryAccessTokenByUser(loginUser, userId);
+        return Result.success(accessTokens);
```
Comment (Radeity) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/AccessTokenController.java:161]: If we do this refactor and only throw exception when meet some unexpected operation, UI will not display alert box with error message, and I think it's not user-friendly.

Comment (ruanwenjun) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/AccessTokenController.java:161]: We have `ApiExceptionHandler` to transform the ServiceException to standard Result, the error message will be set in to Result, this is same with directly return Result.

**Review by SbloodyS — DISMISSED** [review]
LGTM. This is really a big change. Thanks 👍🏻

**Comment by ruanwenjun** [comment]
Resolve the conflicts.

**Review by caishunfeng — APPROVED** [review]
LGTM overall

**Pull request merged**